src/main/java/pages/CentralAreaPage.java

```diff
@@ -1,4 +1,240 @@
 package pages;
 
-public class CentralAreaPage {
+import org.openqa.selenium.By;
```

Comment:
Have only one class to abstract all actions is not good. at least you should have 3 classes, 1 for section, project and task